src/main/java/org/openpnp/gui/AlphaIcon.java

```diff
@@ -0,0 +1,93 @@
+/**
```

Comment:
Hi Jason

No problem, I've checked the licence and put info in the commit comment, but having it in the comment section makes it easier to reference.

Matt